pennylane/ops/qubit.py

```diff
@@ -565,6 +567,96 @@ def decomposition(phi, theta, omega, wires):
         return decomp_ops
 
 
+class PauliRot(Operation):
+    r"""PauliRot(theta, pauli_word, wires)
+    Arbitrary pauli word rotation.
+
+    .. math::
+
+        RP(\theta, P) = \exp(-i \frac{\theta}{2} P)
```

Comment:
Would `PR` make more sense given this naming rather than`RP`? (Let me know, however, if this is coming from some parts of the literature.)

Reply:
I didn't actually look in the literature, but I wanted to be consistent with `RX`, `RY` and `RZ`